packages/vsphere/changelog.yml

```
@@ -1,4 +1,10 @@
 # newer versions go on top
+
+- version: "1.0.0"
+  changes:
+    - description: Making Vsphere GA
```

Nit. This changelog is already for the vsphere package, no need to mention it.
```suggestion
    - description: Release as GA
```